Support multiple input streams

nirmalvp opened the pull request:
Input can be obtained from any stream that implements the ioReader
interface

Commit:
Support multiple input streams

Input can be obtained from any stream that implements the ioReader
interface

nirmalvp committed:
Refactoring

sarathsp06 commented on main.go:
```diff
@@ -74,6 +74,7 @@ func getKey() string {
 
 func main() {
 	key := getKey()
+	// TODO : Support inputs other than stdin
```
Remove this

sarathsp06 commented on input.go:
```diff
@@ -1,18 +1,27 @@
+}
+
+// getStreamData reads input from the input device ,panics if fail
+func getStreamData(inputStream io.ReadCloser) ([]byte, error) {
```
Move this to getInput

nirmalvp committed:
Cleaning up

1) Removed a stale comment
2) Combined function to readstream with the function to get the user
input

sarathsp06 commented on input.go:
```diff
@@ -1,18 +1,23 @@
+	return os.Stdin
+}
+
+// getStreamData reads input from the input device ,panics if fail
```
Fix the comment please
